common/cache-manager.js

```diff
@@ -222,18 +222,7 @@ var cacheManager = {
         this.writeCacheFile(function () {
             function deferredDelete () {
                 var item = caches.pop();
-                if (self._isZipFile(item.originUrl)) {
-                    if (self._isZipFile(item.url)) {
-                        deleteFile(item.url, self._deleteFileCB.bind(self));
-                    }
-                    else {
-                        rmdirSync(item.url, true);
-                        self._deleteFileCB();
-                    }
-                }
-                else {
-                    deleteFile(item.url, self._deleteFileCB.bind(self));
-                }
+                self._removePathOrFile(item.originUrl, item.url);
```

Comment:
Changing "path" to "directory" might be more appropriate.

Reply:
Path is shorter than directory, I prefer to use 'path'